corgidrp/mocks.py

```diff
@@ -1056,9 +1056,12 @@ def create_astrom_data(field_path, filedir=None, image_shape=(1024, 1024), subfi
     prihdr['VISTYPE'] = 'BORESITE'
     prihdr['RA'] = target[0]
     prihdr['DEC'] = target[1]
+    prihdr['ROLL'] = 0   ## assume a telescope roll = 0 for now
```

Comment:
Note that this will be covered by this upcoming PR: https://github.com/roman-corgi/corgidrp/pull/306/files#diff-02971e9165951439a9a69d418600747c90eb4612e412cd077fdab3655c0343eaR150. No action now, but we can remove it in the future.